index.html

```diff
@@ -290,6 +290,28 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer>
+      <div class="form-div">
+        <div class="form-info">
+          <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <form action="https://formspree.io/f/form_id" method="post">
+        <div class="name-wrapper">
+          <div class="input-wrapper first-name-input">
+            <input type="text" id="first-name" placeholder="First Name">
+          </div>
+          <div class="input-wrapper last-name-input">
+            <input type="text" placeholder="Last Name">
+          </div>
+        </div>
+        <div class="email-wrapper">
+          <input type="email" placeholder="Your email">
+        </div>
+        <textarea cols="30" rows="10" placeholder="Your message here..."></textarea>
+        <button type="button" class="btn-get">Get in touch</button>
+      </form>
+      </div>
+    </footer>
```

Comment:
-  Please follow the correct font style to your contact form.
-  Kindly add the correct placeholders according to the Figma Design.
-  You should  align the elements according to the requirement,  You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your project looks like this
![Project-secreenshot](https://user-images.githubusercontent.com/93598189/184003262-fe94c3d8-7185-4742-a5ef-9f6c7dd64d1c.PNG)

Figma Design look like this : 
![Figma design](https://user-images.githubusercontent.com/93598189/184003519-b82e1b1a-7555-49dd-91e7-4bd34c8099a1.PNG)